### ISSUE TITLE
N75 definition and parsing QUAST report

### ISSUE THREAD

**Issue by cyklee** [ewels/MultiQC#487]
Hi Phil,

Just reporting two very minor bugs.

**Description of bug:**
1. Mouseover N75 definition in QUAST output is currently `QUAST: N75 in the contig length such that using longer or equal length contigs produces half (50%) of the bases of the assembly (kilo base pairs)`. This should be 75%. (modules/quast/quast.py line 147)

2. In Linux environment, currently parsing the output directory of metagenomic QUAST (not sure about single genome mode) will result in an error message. I suspect it is because QUAST's unusual file names in the summary directory:
```
For example:
#_N's_per_100_kbp.tsv
#_operons.tsv
#_predicted_genes_(unique).tsv
Total_aligned_length.tsv
Total_length_(>=_10000_bp).tsv
```
I could mitigate this problem by pointing multiQC directly to `report.tsv` in `combined_reference` (```multiqc -d `fzf -m` ``` works really well.) This also saves a lot of time as some of the files in QUAST results are pretty big to scan through.

* **File that triggers the error:**
    * I can place the files in Dropbox if you need them.
* **Operating System:**
    * Debian Jesse in VM
* **Python Version:**
    * 3.6.1
* **Method of MultiQC installation:**
    * pip, multiqc version 1.1.dev0

Thanks and keep up the great work!

**Comment by ewels** [comment]
Thanks for the bug report, I'll take a look into it.. Regarding the Linux bug - what was the error message that you get? Also, are there any fixed filenames that we could skip from the QUAST output to speed up execution? We can add these to the default list of [filenames to ignore](https://github.com/ewels/MultiQC/blob/master/multiqc/utils/config_defaults.yaml#L148-L161)..

Phil

**Comment by ewels** [comment]
Part 1) fixed..

**Comment by ewels** [comment]
Maybe @vladsaveliev may have some input on this issue?

**Comment by vladsaveliev** [comment]
Hi guys,

`metaquast.py`'s output is different from the regular `quast.py`, and I believe that the multiqc module just wasn't designed and tested for meta. I just ran a quick test and had an error `KeyError: '# contigs (>= 0 bp)'`, which doesn't seem to be related to the file names (though I agree that having hashes and parenthesis in file names is a stupid idea - we will change that in quast too!). 

Ok, so, I'll see if I can quickly fix the module to make it work.

Regarding the files to skip, there are long outputs from alignment and large misassembly reports, which could be skipped. I'll take care of that too.

**Comment by ewels** [comment]
Great stuff, thanks! 👍  Let me know if I can be of any help..

Phil

**Comment by ewels** [comment]
Fixed in #491. @cyklee - please let us (or rather, @vladsaveliev) know if you have any further problems..

Phil
